src/resultString.ts

```diff
@@ -0,0 +1,6 @@
+import { CodacyResult } from "./model/codacyResult"
+
+export function resultString(results: CodacyResult[]): string {
+  const lines = results.map(result => JSON.stringify(result))
+  return lines.join("\n")
+}
```

Comment:
You could add a `toString` method to CodacyResult model instead

Reply:
It takes an array of results and not a single result.

Reply:
You're right 👍